### ISSUE TITLE
[BUG] ConvTranspose with out_channels=1 

### ISSUE THREAD

**Issue by mikeseven** [pytorch/pytorch#65661]
https://github.com/pytorch/pytorch/blob/1c8949c51ac425c3e4fef57e296784bea576afdc/torch/nn/modules/conv.py#L85

If ConvTranpose has groups = number of output channels of ungrouped convtranspose then out_channels=1 and this line fails.
It should be out_channels > 1 and out_channels % groups != 0
I believe the same is true for in_channels check just above.

**Comment by gchanan** [comment]
do you have an example of this failing on real code?

**Comment by mikeseven** [comment]
Yes during quantization of centernet dlav0 that has convtransposes with lots of groups and all are equal to the number of output channels, aka depthwise convtranspose.
I added the check in the code, as highlighted and it works, otherwise crash.
